packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -4553,3 +4553,38 @@ QUnit.module('searchStartEvent', {
         }
     });
 });
+
+QUnit.module('accessibility', {
+    beforeEach: function() {
+        this.$lookup = $('#lookup');
+        const defaultOptions = {
+            items: ['1', '11', '111'],
+            value: '1',
+            dropDownOptions: {
```

Comment:
Check the scenario in which you install:

```
dropDownOptions: {
      hideOnOutsideClick: false,
},
```

you will see that Lookup doesn't close before your changes if you click outside of Lookup (see demo: https://codepen.io/marker-dao/pen/WNqWJrq??editors=1010), but with your changes it works differently - Lookup closes if you click outside of it.